src/index.ts

```diff
@@ -5,5 +5,6 @@ export * from './node-process/ProcessParentMessageStream';
 export * from './node-process/ProcessMessageStream';
 export * from './node-thread/ThreadParentMessageStream';
 export * from './node-thread/ThreadMessageStream';
+export * from './runtime/BrowserRuntimePostMessageStream';
```

Comment:
This might fail outside of browsers because of `webextension-polyfill`. So perhaps remove?